Formula/terminator.rb

```diff
@@ -39,7 +39,12 @@ def install
     pid = Process.spawn bin/"terminator", "-d", [:out, :err] => "#{testpath}/output"
     sleep 5
```

Comment:
Not sure about macOS error, but since it happens sporadically (`12` failed and passed on separate runs), maybe try increasing sleep.

Reply:
You were totally right - the issue I guess it that amount of time it takes for `terminator` to start in CI is more variable than it is on my local machine, explaining why I could not reproduce this failure locally.